algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java

```diff
@@ -30,11 +31,9 @@ class ActionableTrialFactory implements ActionableEntryFactory {
 
     private static final Set<String> AGE_GROUPS_TO_INCLUDE = Sets.newHashSet();
 
-
     static {
         POTENTIALLY_OPEN_RECRUITMENT_TYPES.add("recruiting");
```

Comment:
you can also create a set with `Set.of()` and remove the need for a mutable set and the static block

Reply:
done